admin/local/docker/compose/docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.9"
```

Comment:
I have built and tried testing the new code in my development environment at USDF's machine `sdfqserv001` (RedHat 8), and the integration test couldn't be started because of:
```
gapon@sdfqserv001 qserv (tickets/DM-48903) $ qserv up --dashboard-port=25089 --http-frontend-port 4049
/usr/lib/python3.6/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.26.20) or chardet (3.0.4) doesn't match a supported version!
  RequestsDependencyWarning)
yaml: map merge requires map or sequence of maps as the value
gapon@sdfqserv001 qserv (tickets/DM-48903) $ python3 --version
Python 3.6.8
```
Could this be a problem of the Python version?

Reply:
Unrelated I believe -- this `version` is docker compose yaml-format version and not at all Python-related.  The compose `version` directive has been deprecated in compose v2, and was generating a warning.

Reply:
We'll need to do _something_ about your ability to run unit tests before we merge this, though...  It could be that system python 3.6 library suite is now too old for the `qserv` Python command, which means you'd need to set up a conda or venv with something newer to run it outside the build container.  I wonder why it broke for you on this PR, though?

Reply:
I suspect, the problem is with the version of Docker CLI and Docker compose plugin which is much newer in Alma 9 versus what I can use on RedHat 8

Reply:
For the record, the problem was solved by upgrading the local version of the plugin to:
```
dnf info docker-compose-plugin
Last metadata expiration check: 2:53:51 ago on Wed 12 Feb 2025 02:47:20 PM PST.
Installed Packages
Name         : docker-compose-plugin
Version      : 2.27.0
Release      : 1.el8
...
```
This makes the local integration test happy.